ttnn/cpp/ttnn/operations/common/l1_interface_common.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <cstdint>
+#include <iostream>
+#include <optional>
+#include <tuple>
+#include <vector>
+
+#include "impl/buffers/buffer.hpp"
+#include "ttnn/tensor/types.hpp"
```

Comment:
nit: do you need these if you are forward declaring?

Reply:
nope, probably got included automatically somehow, fixed